services/getStudentAttendanceService.go

```diff
@@ -0,0 +1,35 @@
+package services
+
+import (
+	"attendance/repository"
+	"attendance/util"
+	"encoding/json"
+	"net/http"
+
+	"go.uber.org/zap"
+)
+
+type GetStudentAttendanceService interface {
+	GetStudentAttendance(username string, data repository.GetStudentAttendanceJSON, w http.ResponseWriter, r *http.Request)
+}
+
+type GetStudentAttendanceServiceImpl struct {
+	repository repository.Repository
```

Comment:
Name should be Generic where i can put diff methods , GET is a property.

Reply:
Changed required files name to generic names.